catalog/spatial/geocoding/spatial-geocoding-index/pom.xml

```diff
@@ -78,10 +92,17 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Export-Package>org.codice.ddf.spatial.geocoding.index</Export-Package>
+                        <Private-Package>
+                            org.codice.ddf.spatial.geocoding.query,
```

Comment:
Didn't you move everything from .query to .index, which would mean you don't need this private package here anymore?